packages/transition-backend/src/socketServerApp.ts

```diff
@@ -42,6 +42,14 @@ const setupSocketServerApp = async function (server, session) {
     serviceLocator.addService('socketEventManager', new events.EventEmitter());
     allSocketRoutes(serviceLocator.socketEventManager);
 
+    // Enqueue/resume running and pending tasks
+    // FIXME This implies a single server process for a given database. We don't
+    // know if the job is enqueued in another process somewhere and may be
+    // executed twice. For pending jobs, we could have a first run, first serve,
+    // but for jobs in progress, we don't know if they are actively being run or
+    // not
+    await ExecutableJob.enqueueRunningAndPendingJobs(serviceLocator.socketEventManager);
```

Comment:
AS per you comment mention, we could have case where you won't want this. 
It should be behind a parameter setting so we can disable it. Enabled by default. 
(I can foresee in particular people running on their desktop not wanting to have this restart if something went bad